src/types/Personalia.ts

```diff
@@ -5,6 +5,9 @@ interface Personalia {
     fortrolig: boolean;
     strengtFortrolig: boolean;
     skjermet: boolean;
+    barn: Barn[];
 }
 
+export interface Barn extends Personalia {}
```

Comment:
@erlendev Virka litt som at det ikke var noe spesielt med barn når det gjaldt Personopplysninger i backend så jeg tok utgangspunkt i det samme her. Kan endre forsåvidt.

Reply:
Og med spesielt mener jeg forskjell. En person er også et barn.

Reply:
God observasjon. Trenger vi egentlig dette `Barn`-interfacet da, hvis de uansett er helt like?

Reply:
Forsåvidt enig. Det var mer for å få en tydligere definisjon på barn. Men kunne jo egentelig bare typet det slik: `barn: Personalia[]`